lib/components/footer/bottomNavigationTabs.dart

```diff
@@ -0,0 +1,59 @@
+import 'package:flutter/material.dart';
+import 'package:off_top_mobile/login.dart';
+import 'package:off_top_mobile/recordingSession.dart';
+import 'package:off_top_mobile/settings_page.dart';
+
+class Tabs extends StatelessWidget {
```

Comment:
Have the class name match the name of the file

Reply:
Done! Completely missed that, my bad.